src/platform/plugins/shared/dashboard/public/services/dashboard_query_client.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { QueryClient } from '@tanstack/react-query';
+
+export const dashboardQueryClient = new QueryClient({
```

Comment:
Creating this client allows to re-use favorites cache between landing and dashboard pages